src/main/java/de/fraunhofer/aisec/crymlin/connectors/db/TraversalConnection.java

```diff
@@ -33,15 +33,6 @@ public Graph getGraph() {
 	}
 
 	public void close() {
-		try {
-			if (tg != null) {
-				tg.close();
-			}
-		}
-		catch (Exception e) {
-			log.debug("Closing TG: ", e);
-		}
-
```

Comment:
While working on #32 and in a sense #95, I ran into a couple of problems with the database. This seems to be the better solution.